src/zcl/definition/cluster.ts

```diff
@@ -4023,6 +4023,14 @@ const Cluster: {
             }
         },
     },
+    manuSpecificTuyaSwitch: {
```

Comment:
Can you rename this to `manuSpecificTuya_3` (I expect this cluster will be used for more than just a switch)